armonik/external-data.tf

```
@@ -38,10 +45,17 @@ locals {
     ip       = module.admin_gui_endpoint.host
     app_port = try(module.admin_gui_endpoint.ports[0], null)
   }
-  admin_old_gui_endpoints = {
-    ip       = module.admin_old_gui_endpoint.host
-    api_port = try(module.admin_old_gui_endpoint.ports[0], null)
-    app_port = try(module.admin_old_gui_endpoint.ports[1], null)
+  # Deprecated, must be removed in a future version
+  admin_0_8_gui_endpoints = {
+    ip       = module.admin_0_8_gui_endpoint.host
+    api_port = try(module.admin_0_8_gui_endpoint.ports[0], null)
+    app_port = try(module.admin_0_8_gui_endpoint.ports[1], null)
+  }
+  # Deprecated, must be removed in a future version
+  admin_0_9_gui_endpoints = {
+    ip       = module.admin_0_9_gui_endpoint.host
+    api_port = try(module.admin_0_9_gui_endpoint.ports[0], null)
+    app_port = try(module.admin_0_9_gui_endpoint.ports[1], null)
```

I think 0.9 has only the app port:
```suggestion
  admin_0_9_gui_endpoints = {
    ip       = module.admin_0_9_gui_endpoint.host
    port = try(module.admin_0_9_gui_endpoint.ports[0], null)
```

---

You're right!